core/dbt/tests/util.py

```diff
@@ -160,6 +161,11 @@ def write_file(contents, *paths):
         fp.write(contents)
 
 
+def file_exists(*paths):
+    """Check if file exists at path"""
+    return os.path.exists(os.path.join(*paths))
+
+
```

Comment:
Not a problem to solve in this PR, but I wonder why `os.path` was used  in this module rather than `pathlib`?

Reply:
Yep, going with the flow make sense in this PR.

Something for us to consider in a refactor down the road though.